src/tests/ftest/pool/evict_test.py

```diff
@@ -20,128 +20,258 @@
   provided in Contract No. B609815.
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
-'''
+"""
 from __future__ import print_function
 
-import os
-import traceback
-
-from avocado.utils import process
 from apricot import TestWithServers
+from daos_api import DaosApiError
+from general_utils import get_container
+from test_utils import TestPool
+from conversion import c_uuid_to_str
+import ctypes
+import uuid
+
 
-class EvictTest(TestWithServers):
+class EvictTests(TestWithServers):
     """
+
     Tests DAOS client eviction from a pool that the client is using.
 
     :avocado: recursive
     """
 
-    def test_evict(self):
+    def connected_pool(self, hostlist, targets=None):
+        """Connect to pool.
+
+        Args:
+            hostlist (list): list of daos server nodes
+            targets (list): List of targets for pool create
+        Returns:
+            TestPool (object)
+
         """
-        Test evicting a client to a pool.
+        pool = TestPool(self.context, self.log)
+        pool.get_params(self)
+        if targets is not None:
+            pool.target_list.value = targets
+        # create pool
+        pool.create()
+        # Check that the pool was created
+        status = pool.check_files(hostlist)
+        if not status:
+            self.log.error("Invalid pool - pool data not detected on servers")
```

Comment:
Call `self.fail(<msg>)` here instead of returning None which will cause problems on lines 85-87.